src/components/Carousel/Carousel.tsx

```diff
@@ -246,25 +270,68 @@ export const Carousel: FC<CarouselProps> = React.forwardRef(
 
     const handleOnWheel = (
       apiObj: scrollVisibilityApiType,
-      ev: React.WheelEvent
+      event: React.WheelEvent
     ): void => {
-      const isTouchpad = Math.abs(ev.deltaX) !== 0 || Math.abs(ev.deltaY) < 15;
+      const isTouchpad =
+        Math.abs(event.deltaX) !== 0 || Math.abs(event.deltaY) < 15;
```

Comment:
curious about this evaluation assignment. whats the presumption here? If they are moving in more than one direction simultaneously then it must be touch?

Reply:
It's a distance threshold used to determine mouse vs trackpad, more info here as it's an old trick for hardware detection --> https://stackoverflow.com/questions/10744645/detect-touchpad-vs-mouse-in-javascript/62415754#62415754